src/CdrWriter.ts

```diff
@@ -166,6 +163,80 @@ export class CdrWriter {
     return this;
   }
 
+  /** Writes the delimiter header returning the endianness flag and object size
+   * NOTE: changing endian-ness with a single CDR message is not supported
+   */
+  dHeader(objectSize: number): CdrWriter {
```

Comment:
Could've been nice to name these `delimiterHeader` and `memberHeader` or whatever the full names are - for readability.